build.gradle

```diff
@@ -273,9 +273,9 @@ subprojects { subproject ->
         testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: powermockVersion
         testImplementation group: 'org.powermock', name: 'powermock-module-junit4', version: powermockVersion
 
-        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.9'
+        testImplementation group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.11-CCD-358'
```

Comment:
Is this going to be replaced with a stable version of the lib?

Reply:
Yes, currently using pre-release. Once QAs are happy, we can release test definitions and replace.